Assets/Mirror/Runtime/Messages.cs

```diff
@@ -215,144 +133,52 @@ public struct SpawnMessage : IMessageBase
         /// <remark>ArraySegment to avoid unnecessary allocations</remark>
         /// </summary>
         public ArraySegment<byte> payload;
-
-        public void Deserialize(NetworkReader reader)
-        {
-            netId = reader.ReadPackedUInt32();
-            isLocalPlayer = reader.ReadBoolean();
-            isOwner = reader.ReadBoolean();
-            sceneId = reader.ReadPackedUInt64();
-            if (sceneId == 0)
```

Comment:
should we add extension methods for this so that it doesn't always write `Guid` now? I think `Guid` will always allocate a new array?

Reply:
we can,  but I don't think it is worth the trouble.  Maybe separate PR?